examples/ensure-name-substring/advanced/README.md

```diff
@@ -43,5 +58,8 @@ Check all resources have `prod-` in their names and the field `.spec.name` in
 `MyResource` also got updated.
 
 ```sh
-kpt cfg cat advanced
+kpt pkg cat advanced
```

Comment:
Cat is deprecated. https://kpt.dev/reference/pkg/

Reply:
Is there a replacement?
Any suggestion?

Reply:
linux cat specific files ?